src/index.html

```diff
@@ -8,6 +8,50 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Stars</h1>
+    <h1></h1>
+    <div class="container">
+      <div class="stars--0">
+        <div></div>
```

Comment:
```suggestion
        <div class='stars__star'></div>
```
Такой класс должен быть у каждой звезды